docs/reference/architecture.md

```diff
@@ -24,7 +24,9 @@ mirrord is composed of the following components:
 * `mirrord-layer` - Rust dynamic library (so/dylib) that loads to the local process, hooks its filesystem, network APIs and relays them to the agent.
 * `mirrord-cli` - Rust binary that wraps the behavior of mirrord-layer in a user friendly CLI.
 * `VS Code extension` - Exposes the same functionality as - mirrord-cli within the VS Code IDE.
-* `IntelliJ plugin` - Exposes the same functionality as - mirrord-cli within the IntelliJ IDEs. ![mirrord - Architecture](architecture/architecture.svg)
+* `IntelliJ plugin` - Exposes the same functionality as - mirrord-cli within the IntelliJ IDEs. 
+
+![mirrord - Architecture](architecture/architecture.svg)
```

Comment:
What's this change for?

Reply:
<img width="877" height="421" alt="image" src="https://github.com/user-attachments/assets/297d6a37-017e-4eb5-add0-ffd3552443ab" />

this is what it looks like currently

Reply:
<img width="897" height="644" alt="image" src="https://github.com/user-attachments/assets/e0c093c3-25bd-47fa-8f52-cd9d9ec454ac" />

currently the architecture diagram is shrunk because the image is on the same line as the bullet. this change makes it the normal readable size